recipe/build.sh

```diff
@@ -1,5 +1,25 @@
-set -ex
+set -o xtrace -o nounset -o pipefail -o errexit
 
-cargo install --path . --root ${PREFIX}
+# Required for cross-compiling with pkg-config
+export PKG_CONFIG_SYSROOT_DIR=$PREFIX
+export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
+export CARGO_BUILD_RUSTFLAGS="$CARGO_BUILD_RUSTFLAGS -L all=$PREFIX/lib"
+
+if [[ "$c_compiler" == "clang" ]]; then
+  echo "-L$BUILD_PREFIX/lib -Wl,-rpath,$BUILD_PREFIX/lib" > $BUILD_PREFIX/bin/$BUILD.cfg
+  echo "-L$PREFIX/lib -Wl,-rpath,$PREFIX/lib" > $BUILD_PREFIX/bin/$HOST.cfg
+fi
+
+# build statically linked binary with Rust
+cargo install --path . --root ${PREFIX} --verbose --locked
 
 cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+
+# strip debug symbols
+"$STRIP" "$PREFIX/bin/cargo-cinstall"
+"$STRIP" "$PREFIX/bin/cargo-cbuild"
```

Comment:
is there no flag in rust to do this automatically?

Reply:
I have no idea. I'm just copying from the other feedstock mentioned above. A quick google search suggests that you need to add flags for the c-linker, not rust. (ldd for linux).

https://users.rust-lang.org/t/compile-in-release-mode-without-debugging-symbols/19764